packages/base/any/kernels/5.10-lts/patches/0012-accton-as4224.patch

```diff
@@ -81,33 +81,37 @@ index 000000000..d38b19668
 +			It is currently not required for Amazon 'ethtool -m' support but it is intended for future use.
 +			Can be skipped in this stage.
 +		*/
-+		los-gpio = <&gpio_i2c 0 GPIO_ACTIVE_HIGH>;
++		los-gpio = <&gpio_i2c 0 GPIO_ACTIVE_LOW>;
```

Comment:
Please make this change in a separate commit with the appropriate description.

Reply:
Already added this changes in a separate commit. Please help check.

Reply:
Sorry, I must have missed it. Can you explain again, why this is changed, that means, what problem is fixed?

Reply:
It fixes wrong CPLD implementation. This is the RX_LOSS signal from SFP towards the CPLD. These changes were confirmed on privet branch with Marvell Switchdev drivers + Accton AS5114 platforms.